<launch> 
    <!-- Turtlebot3 Gazebo -->
  <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_house.launch">
    <arg name="model" value="waffle_pi"/>
    <arg name="x_pos" value="-2.0"/>
  </include>

    <!-- Turtlebot3 Navigation -->
  <include file="$(find turtlebot3_navigation)/launch/move_base.launch">
    <arg name="model" value="waffle_pi"/>
  </include>

    <!-- Turtlebot3 SLAM -->
  <include file="$(find turtlebot3_slam)/launch/turtlebot3_slam.launch">
    <arg name="slam_methods" value="gmapping" />
    <arg name="open_rviz"    value="true" />
  </include>
  
  <node name="detect_ball" pkg="robotics_challenge" type="detect_ball.py" output="screen"/>
</launch>
